nilrt_snac/_configs/_auditd_config.py

```diff
@@ -77,8 +77,11 @@ class _AuditdConfig(_BaseConfig):
     def __init__(self):
         super().__init__("auditd")
         self._opkg_helper = opkg_helper
-        self.log_path = os.path.realpath("/var/log")
-        self.audit_config_path = "/etc/audit/auditd.conf"
+        self.log_path = pathlib.Path("/var/log")
+        self.audit_config_path = pathlib.Path("/etc/audit/auditd.conf")
+        self.audit_rule_script_path = pathlib.Path("/etc/audit/audit_email_alert.pl")
```

Comment:
Consider: I know this is based on the variable name it had below, but I just felt the name was a little off. To me either `audit_email_rule_path` or `audit_email_script_path` would seem like a better name. Feel free to override me on this one.